map/lanelet2_extension/lib/query.cpp

```diff
@@ -40,7 +40,15 @@ double getAngleDifference(const double angle1, const double angle2)
   Eigen::Vector2d vec1, vec2;
   vec1 << std::cos(angle1), std::sin(angle1);
   vec2 << std::cos(angle2), std::sin(angle2);
-  const double diff_angle = std::acos(vec1.dot(vec2));
+  const double inner_product = vec1.dot(vec2);
+  double diff_angle;
+  if (inner_product < -1.0) {
+    diff_angle = M_PI;
+  } else if (inner_product > 1.0) {
+    diff_angle = 0.0;
+  } else {
+    diff_angle = std::acos(inner_product);
+  }
   return std::fabs(diff_angle);
 }
```

Comment:
I guess this function can be replaced by a function like [this](https://github.com/autowarefoundation/autoware.universe/blob/2b1c1b329bcca7070934ff26e2cebc3e4ed3795f/common/tier4_autoware_utils/include/tier4_autoware_utils/math/normalization.hpp#L36-L46).

Also, since this is a library, adding some tests is better.
If you can't work on the task right now, creating a follow-up issue is fine.

Reply:
Replacing that function is suggedted also by ohta san but found the solution so I did't examine well...
But It seems better that using comomon function so i replace with that.

And also I will add tests for getAngleDifference, but still there are many functions which don't have test in the file so I will create a follow-up issue.

Reply:
If you use `autoware_utils`, I think adding tests isn't mandatory because it's already tested, but you should be aware that it adds a dependency.